keys/keys.go

```diff
@@ -121,9 +121,15 @@ func RangeStatsKey(rangeID roachpb.RangeID) roachpb.Key {
 
 // SequenceCacheKey returns a range-local key by Range ID for a
 // sequence cache entry, with detail specified by encoding the
-// supplied client command ID.
-func SequenceCacheKey(rangeID roachpb.RangeID, family []byte) roachpb.Key {
-	return MakeRangeIDKey(rangeID, LocalSequenceCacheSuffix, encoding.EncodeBytes(nil, family))
+// supplied transaction ID and sequence number.
+// The length of the prefix of the encoded key excluding the encoded
+// sequence number is returned.
+func SequenceCacheKey(rangeID roachpb.RangeID, id []byte, seq uint32) (roachpb.Key, int) {
```

Comment:
I don't see any case where you need both: the only time you use the prefixLen return value you're passing in a dummy sequence number to use as the starting point of the scan. That scan would work just as well with the prefix alone.